pages/configuring_webif.rst

```diff
@@ -308,106 +308,127 @@ If you want to run a load balancer/reverse proxy in front of Graylog, you need t
   - You are either using only HTTP or only HTTPS (no mixed content) for both the web interface endpoint and the REST API endpoint.
   - If you use SSL, your certificates must be valid and trusted by your clients.
 
-To help you with your specific environment, here are some example configurations for common scenarios:
+.. NOTE:: To help you with your specific environment, we have some example configurations. We take the following assumption in all examples. Your Graylog server.conf has the following settings set ``rest_listen_uri = http://127.0.0.1:12900/`` and ``web_listen_uri = http://127.0.0.1:9000/``. Your URL will be ``graylog.example.org`` with the IP ``192.168.0.10``.
 
 
 Using a Layer 3 load balancer (forwarding TCP Ports):
 -----------------------------------------------------
 
-For the following example we're assuming that your Graylog server is running on IP 1.2.3.4. Your external IP (the one external clients are using to access the Graylog instance) is 2.3.4.5.
-
-#. Configure your load balancer to forward connections going to ``2.3.4.5:80`` to ``1.2.3.4:9000`` and ``2.3.4.5:12900`` to ``1.2.3.4:12900``.
-#. Set ``web_endpoint_uri`` in your Graylog server config to ``http://2.3.4.5:12900``.
+#. Configure your load balancer to forward connections going to ``192.168.0.10:80`` to ``127.0.0.1:9000`` and ``192.168.0.10:12900`` to ``127.0.0.1:12900``.
+#. Set ``web_endpoint_uri`` in your Graylog server config to ``http://graylog.example.org:12900``.
 #. Start the Graylog server as usual
-#. Access the web interface on ``http://2.3.4.5``.
+#. Access the web interface on ``http://graylog.example.org``.
 #. Read up on :ref:`ssl_setup`.
 
 NGINX:
 ------
 
-For the following samples we are assuming that your Graylog instance is running on ``graylog.internal.example.org`` using the default ports of 12900 for the REST API and 9000 for the web interface. SSL is disabled for both. You want to expose the Graylog web interface as ``https://graylog.example.org``. The configuration for TLS certificates, keys and ciphers is omitted from the sample config for brevity's sake.
-
-If you want to use nginx to proxy access to a Graylog server, you have several options:
+**REST API and Web Interface on one port (using HTTP)**::
 
-**REST API and Web Interface on one port (using HTTPS/SSL)**::
+    server
+    {
+      listen      80 default_server;
+      listen      [::]:80 default_server ipv6only=on;
```

Comment:
Is this `listen` directive really required? Doesn't the first one bind to an IPv4 *and* IPv6 address?